packages/zowe-explorer/__tests__/__unit__/utils/TreeViewUtils.unit.test.ts

```diff
@@ -23,4 +31,155 @@ describe("TreeViewUtils Unit Tests", () => {
         listenerFn({ element });
         expect(testTreeProvider.mOnDidChangeTreeData.fire).toHaveBeenCalledWith(element);
     });
+
+    describe("errorForUnsavedResource", () => {
+        function getBlockMocks(): Record<string, jest.SpyInstance> {
+            return {
+                errorMessage: jest.spyOn(Gui, "errorMessage").mockClear(),
+                profilesInstance: jest.spyOn(Profiles, "getInstance").mockReturnValue({
+                    loadNamedProfile: jest.fn().mockReturnValue(createIProfile()),
+                    checkCurrentProfile: jest.fn(),
+                } as any),
+            };
+        }
+        it("prompts for an unsaved USS file", async () => {
+            const ussNode = new ZoweUSSNode({
+                label: "testFile.txt",
+                collapsibleState: vscode.TreeItemCollapsibleState.None,
+                profile: createIProfile(),
+                parentNode: createUSSSessionNode(createISession(), createIProfile()),
+            });
+            const blockMocks = getBlockMocks();
+
+            const textDocumentsMock = jest.replaceProperty(vscode.workspace, "textDocuments", [
+                {
+                    fileName: ussNode.getUSSDocumentFilePath() as any,
+                    uri: vscode.Uri.file(ussNode.getUSSDocumentFilePath()),
+                    isDirty: true,
+                } as any,
+            ]);
+
+            expect(await TreeViewUtils.errorForUnsavedResource(ussNode)).toBe(true);
+            expect(blockMocks.errorMessage).toHaveBeenCalledWith(
+                "Unable to rename testFile.txt because you have unsaved changes in this file. " + "Please save your work and try again.",
```

Comment:
The `+` sign seems unnecessary here since the text isn't split across multiple lines:
```suggestion
                "Unable to rename testFile.txt because you have unsaved changes in this file. Please save your work and try again.",
```